npm tossing two errors on linux (one related to ELIFECYCLE and one related to script references)

[2020-06-02T02_56_21_194Z-debug.log](https://github.com/daniel-sc/bash-shell-to-bat-converter/files/4714489/2020-06-02T02_56_21_194Z-debug.log)

When running "npm run tsc" it spits out the following: 

```
> bash-converter@0.1.0 tsc /home/dbz/github/bash-shell-to-bat-converter
> tsc

error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/e2e/src/app.e2e-spec.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/e2e/src/app.po.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/src/app/app.component.spec.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/src/app/app.component.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/src/app/app.module.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/src/environments/environment.prod.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/src/environments/environment.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/src/main.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/src/polyfills.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
error TS6059: File '/home/dbz/github/bash-shell-to-bat-converter/ui/src/test.ts' is not under 'rootDir' '/home/dbz/github/bash-shell-to-bat-converter/src'. 'rootDir' is expected to contain all source files.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! bash-converter@0.1.0 tsc: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the bash-converter@0.1.0 tsc script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dbz/.npm/_logs/2020-06-02T02_56_21_194Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -9,7 +9,7 @@ Try it online: https://daniel-sc.github.io/bash-shell-to-bat-converter/
 ## Usage
 Setup:
 ```sh
-git clone git@github.com:daniel-sc/bash-shell-to-bat-converter.git
+git clone https://github.com/daniel-sc/bash-shell-to-bat-converter.git
 cd bash-shell-to-bat-converter
 npm install
 npm run tsc

--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
   "name": "bash-converter",
   "version": "0.1.0",
   "description": "converting simple bash/shell scripts to windows batch files",
+  "private": true,
   "main": "lib/cli.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -24,7 +24,7 @@
 
     /* Strict Type-Checking Options */
     "strict": true,                           /* Enable all strict type-checking options. */
-    // "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
+    "noImplicitAny": false,                   /* Raise error on expressions and declarations with an implied 'any' type. */
     // "strictNullChecks": true,              /* Enable strict null checks. */
     // "strictFunctionTypes": true,           /* Enable strict checking of function types. */
     // "strictBindCallApply": true,           /* Enable strict 'bind', 'call', and 'apply' methods on functions. */
@@ -61,5 +61,8 @@
 
     /* Advanced Options */
     "resolveJsonModule": true                 /* Include modules imported with '.json' extension */
-  }
+  },
+  "exclude": [                                /* Fixes https://github.com/daniel-sc/bash-shell-to-bat-converter/issues/7 */
+    "ui"
+  ]
 }

